lib/edmunds/vehicle/specification/color.rb

```diff
@@ -10,11 +10,16 @@ module Specification
       module Color
         
         class ColorsByStyle
-          attr_reader :colors, :count
+          attr_reader :exterior, :interior #, :colors, :count
 
           def initialize(attributes)
-            @colors = attributes['colors'].map { |json| ColorsDetails.new(json) } if attributes.key?('colors')
-            @count = attributes['colorsCount']
+            # @colors = attributes['colors'].map { |json| ColorsDetails.new(json) } if attributes.key?('colors')
```

Comment:
I'm not sure if it was different in the past. Right now doing .map over colors just loops over a couple of keys (interior and exterior) not the actual colors.